docs/addons/ack-addon.md

```diff
@@ -149,5 +149,12 @@ Please refer to following aws-controller-8s references for more information :
 25. SQS
 26. SAGEMAKER
 27. EVENTBRIDGE
+28. PIPES
+29. SECRETSMANAGER

```

Comment:
Similar comment: let's order the list so that customer are not confused and can easily identify which service is supported.